ignition/modules/Lock.js

```diff
@@ -1,16 +1,16 @@
 // This setup uses Hardhat Ignition to manage smart contract deployments.
 // Learn more about it at https://hardhat.org/ignition
 
-const { buildModule } = require("@nomicfoundation/hardhat-ignition/modules");
+const { buildModule } = require('@nomicfoundation/hardhat-ignition/modules');
 
 const JAN_1ST_2030 = 1893456000;
 const ONE_GWEI = 1_000_000_000n;
 
-module.exports = buildModule("LockModule", (m) => {
-  const unlockTime = m.getParameter("unlockTime", JAN_1ST_2030);
-  const lockedAmount = m.getParameter("lockedAmount", ONE_GWEI);
+module.exports = buildModule('LockModule', (m) => {
+  const unlockTime = m.getParameter('unlockTime', JAN_1ST_2030);
```

Comment:
What does this mean?

Reply:
I believe it is a time-based lock, that controls actions allowed based on an expiration date.
Since we currently use phases, I believe this is not necessary for now

Reply:
> What does this mean?

I believe it is a time-based lock, that controls actions allowed based on an expiration date.
Since we currently use phases, I believe this is not necessary for now